setup.py

```diff
@@ -414,6 +415,37 @@ def has_html(self):
     sub_commands.append(('install_html', has_html))
 
 
+class CheckTestsErrors(Command):
+    """ Checks tests summary.json for failed tests and raises errror if found """
+    description = "Checks tests summary.json for failed tests and raises errror if found"
+    
+    user_options = [
+        ("summary-file=",None,"Path to summary.json"),
+        ("ignore-gevent-errors",None,"Failed tests using GreenMode.Gevent will not raise error."),
+    ]
+    
+    def initialize_options(self):
+        self.summary_file = "summary.json"
+        self.ignore_gevent_errors = False
+    
+    def finalize_options(self):
+        if self.ignore_gevent_errors is not False:
+            self.ignore_gevent_errors = True
+        if not os.path.isfile(self.summary_file):
+            raise Exception("Provided summary.json file does not exists.")
+            
+    def run(self):
+        with open(self.summary_file,"r") as f:
+            summary = f.read()
+            summary = json.loads(summary)
```

Comment:
This is fine, but note that `json.load` will read and load from the file in one step (instead of a `read` and `loads`).